source/site-administrators/plugin-management.rst

```diff
@@ -113,6 +113,10 @@ For more information on the API, see :studio_swagger_url:`#/marketplace/installP
 Managing Installed Plugins
 --------------------------
 
+^^^^^^^^^^^^^^^^^^^^^^
+View Installed Plugins
+^^^^^^^^^^^^^^^^^^^^^^
+
 To view the installed plugins in your site, open the ``Sidebar``.  Click on ``Site Tools``, then ``Plugin Management``.
```

Comment:
Let's start telling people to use the launcher on the top right. Click the launcher on the top right and click on Site Tools.